### PR TITLE
Fix kick and mute permission, fix jumpto settings not working

### PR DESCRIPTION
**Pull request by PonyPC**

### CODE REVIEW

File: bansystem-lib/src/main/java/ch/dkrieger/bansystem/lib/command/defaults/KickCommand.java [hunk 1]
```diff
@@ -43,6 +43,12 @@ public KickCommand() {
     }
     @Override
     public void onExecute(NetworkCommandSender sender, String[] args) {
+        if(!sender.hasPermission(reason.getPermission())&& !sender.hasPermission("dkbans.*")){
+            sender.sendMessage(Messages.REASON_NO_PERMISSION
+                    .replace("[prefix]",getPrefix())
+                    .replace("[reason]",reason.getDisplay()));
+            return;
+        }
```
Comment (DevKrieger) [inline bansystem-lib/src/main/java/ch/dkrieger/bansystem/lib/command/defaults/KickCommand.java:51]: "reason" variable is not available at this point. Which permission is not working?
Permission is already checked in command manager.

File: bansystem-lib/src/main/java/ch/dkrieger/bansystem/lib/command/defaults/MuteCommand.java [hunk 2]
```diff
@@ -41,7 +41,7 @@ public MuteCommand() {
     @Override
     public void onExecute(NetworkCommandSender sender, String[] args) {
         if(BanSystem.getInstance().getConfig().banMode == BanMode.SELF){
-            if(!sender.hasPermission(reason.getPermission())&& !sender.hasPermission("dkbans.*")){
+            if(!sender.hasPermission(this.permission)&& !sender.hasPermission("dkbans.*")){
                 sender.sendMessage(Messages.REASON_NO_PERMISSION
                         .replace("[prefix]",getPrefix())
                         .replace("[reason]",reason.getDisplay()));
```
Comment (DevKrieger) [inline bansystem-lib/src/main/java/ch/dkrieger/bansystem/lib/command/defaults/MuteCommand.java:47]: reason variable is not available
this.permission will be null
Wrong permission message

File: bansystem-lib/src/main/java/ch/dkrieger/bansystem/lib/command/defaults/TempmuteCommand.java [hunk 3]
```diff
@@ -40,6 +40,12 @@ public TempmuteCommand() {
     }
     @Override
     public void onExecute(NetworkCommandSender sender, String[] args) {
+        if(!sender.hasPermission(reason.getPermission())&& !sender.hasPermission("dkbans.*")){
+            sender.sendMessage(Messages.REASON_NO_PERMISSION
+                    .replace("[prefix]",getPrefix())
+                    .replace("[reason]",reason.getDisplay()));
+            return;
+        }
         if(args.length < 3 || !(GeneralUtil.isNumber(args[1]))){
```
Comment (DevKrieger) [inline bansystem-lib/src/main/java/ch/dkrieger/bansystem/lib/command/defaults/TempmuteCommand.java:49]: Permission is already checked in the command manager.

**Comment by PonyPC** [comment]
Am I right now?

File: bansystem-lib/src/main/java/ch/dkrieger/bansystem/lib/command/defaults/KickCommand.java (hunk 1 above)
Comment (PonyPC) [reply bansystem-lib/src/main/java/ch/dkrieger/bansystem/lib/command/defaults/KickCommand.java:51]: You can try normal user can use kick and mute command.

**Comment by DevKrieger** [comment]
Hi

Why do you check the permission in the KickCommand? the permission is already defined in the constructor and is checked by the command manager.

**Comment by DevKrieger** [comment]
No response

**Pull request closed without merging**